### ISSUE TITLE
'React/RCTEventEmitter.h' file not found

### ISSUE THREAD

**Issue by dakkafex** [transistorsoft/react-native-background-geolocation#775]
## Your Environment
* Plugin version: 3.0.9
* Platform: iOS
* OS version: 12
* Device manufacturer / model: XR
* React Native version (`react-native -v`): 0.59.8
* Plugin config

## Expected Behavior

Build to succeed 

## Actual Behavior

build fails with the error:
``` 
'React/RCTEventEmitter.h' file not found
```
## Steps to Reproduce

1. install plugin with pod instructions
2. start build
3. podfile:
```
# Uncomment the next line to define a global platform for your project
platform :ios, '9.0'
# Required by RNFirebase
pod 'Firebase/Core', '~> 5.20.1'

target 'WhosThat_app' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!
  pod 'Firebase/Firestore'
  pod 'Firebase/Storage'
  pod 'Firebase/Auth'
  pod 'GoogleMaps', '~> 2.7.0'
  pod 'Fabric', '~> 1.10.1'
  pod 'Crashlytics', '~> 3.13.1'
  # Pods for WhosThat_app
  pod 'RNImageCropPicker', :path => '../node_modules/react-native-image-crop-picker'
  
  pod 'RNBackgroundGeolocation', :path => '../node_modules/react-native-background-geolocation'
  
  pod 'RNBackgroundFetch', :path => '../node_modules/react-native-background-fetch'
  
  pod 'RNDeviceInfo', :path => '../node_modules/react-native-device-info'

  target 'WhosThat_appTests' do
    inherit! :search_paths
    # Pods for testing
  end

end

```

## Context

trying to install the plugin for the iOS app
## Debug logs

<details><summary>Logs</summary>

``` <!-- Syntax highlighting:  DO NOT REMOVE -->
PASTE_YOUR_LOGS_HERE
```

</details>

**Comment by christocracy** [comment]
```
$ cd ios
$ pod install
```

**Comment by dakkafex** [comment]
Tried many things, among that, i did get a bit further? i am now getting the error 
``'folly/Portability.h' file not found`` im not sure it is related to this plugin (and yes i have added pod 'Folly', podspec: '../node_modules/react-native/third-party-podspecs/Folly.podspec' to my podfile).

but just running pod install did not fix the issue

**Comment by christocracy** [comment]
Your problems are not related to this plugin.

**Comment by dakkafex** [comment]
I tried reverting back to a commit where my app would build on Xcode, I then proceeded to do the following steps:

1.  yarn add react-native-background-geolocation
2.  yarn add cocoa-lumberjack
3.  react-native link react-native-background-geolocation
4.  react-native link react-native-background-fetch
5.  react-native link cocoa-lumberjack
6. pod install

then I try to build, and it fails with an error, in the ``RNBackgroundFetch`` library ``React/RCTEventEmitter.h' file not found``
